config/database.yml

```diff
@@ -1,20 +1,24 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  host: db
-  username: postgres
-  password:
+  host: <%= ENV['DATABASE_HOST'] %>
+  username: <%= ENV['DATABASE_USERNAME'] %>
+  password: <%= ENV['DATABASE_PASSWORD'] %>
   pool: 5
 
-production:
-  <<: *default
-  database: hello_world_production
-
 development:
   <<: *default
-  database: hello_world_development
-
+  database: <%= ENV['DATABASE_DEVELOPMENT'] %>
 
 test:
   <<: *default
-  database: hello_world_test
\ No newline at end of file
+  database: <%= ENV['DATABASE_TEST'] %>
+
+production:
+  <<: *default
+  database: <%= ENV['DATABASE_PRODUCTION'] %>
+  host: <%= ENV['DATABASE_HOST_PRODUCTION'] || ENV['DATABASE_HOST'] %>
```

Comment:
Why not just ENV['DATABASE_HOST']? the same for other env keys? Why do we need "_PRODUCTION" affix here?